kubebeat/beater/evaluation_result_parser.go

```diff
@@ -37,6 +40,11 @@ func (parser *evaluationResultParser) ParseResult(result interface{}, uuid uuid.
 				"rule":     finding.Rule,
 			},
 		}
+		// Insert datastream as raw index to event
+	if index != "" {
+		event.Meta = common.MapStr{libevents.FieldMetaRawIndex: index}
```

Comment:
```suggestion
		event.Meta = common.MapStr{libevents.FieldMetaIndex: index}
```
Try that as it's supposed to take precedence over `FieldMetaRawIndex`

Reply:
OK for now, to be fixed in a later PR.